hack/unwanted-dependencies.json

```diff
@@ -153,6 +153,10 @@
         "sigs.k8s.io/kustomize/api",
         "sigs.k8s.io/kustomize/kustomize/v5"
       ],
+      "golang.org/x/exp": [
+        "github.com/google/cel-go",
+        "github.com/antlr/antlr4/runtime/Go/antlr/v4"
+      ],
```

Comment:
why does the order matter?

Reply:
The array is not in order as well. 

In JSON, the map key may not be in order. But the array is also not in order. It will make the logic of verification harder.

I prefer to make it in order. I also can try to make it check without an order which makes it harder to find what is missing.

Reply:
The fix is something like https://github.com/kubernetes/kubernetes/compare/master...pacoxu:unwanted-status-order-2?expand=1. 

@dims which do you prefer?

Reply:
@pacoxu either way is fine, if it needs to be in-order we need to enforce it somehow (or document it)

Reply:
I prefer to make it in-order. 

I added some comments in the shell.